### ISSUE TITLE
Is the highstock-release dependency necessary for Simple highcharts?

### ISSUE THREAD

**Issue by appsmehta** [whawker/react-jsx-highcharts#5]
@whawker  Do we need to have highstock for absolutely everything? 

I feel it is better to allow simple highcharts to be rendered without highstock. This is because for commercial uses a developer may not have license to highstocks, even though they have purchased a highcharts license.

**Comment by whawker** [comment]
Hmm interesting, I hadn't thought about the licencing issue.

In the short term, if you're using Webpack, I think you should be able to use [`resolve.alias`](https://webpack.js.org/configuration/resolve/#resolve-alias), to use Highcharts instead.
```
resolve: {
  alias: {
    'highstock-release': 'highcharts'
  }
}
```
Could you let me know if the above works? (If you are using Webpack)

In the meantime, I'll look into using [optional](https://www.npmjs.com/package/optional) requires, falling back to requiring `highcharts` if `highstock-release` is not present. Means I probably need to make Highstock an `optionalDependency` rather than a `peerDependency`, I don't know what the longer term impact of that might be though.

In the long term, it might be best if we offer two different NPM packages, one for Highcharts, one for Highstock.

**Comment by appsmehta** [comment]
@whawker , Actually it would still have difficulty in running this library with simple boiler-plate starters like create-react-app. One cannot get webpack without ejecting from create-react-app.

**Comment by whawker** [comment]
Fair enough, do you have a preference on the second or third options?

**Comment by whawker** [comment]
@appsmehta just published version 1.0.0 - we now have separate NPM packages for Highcharts and Highstock. This will allow you to use `highcharts` instead of `highstock-release`.

More details in the [release notes](https://github.com/whawker/react-jsx-highcharts/releases/tag/1.0.0)
